force-app/main/default/classes/OppUpdater.cls

```diff
@@ -0,0 +1,12 @@
+public class OppUpdater {
+    Public Static Void GetAndUpdate(){
```

Comment:
```suggestion
    public static void getAndUpdate(){
```
Careful with the casing/naming convention. It looks like a lot of the PMD errors below are from having variable and method names that begin with an uppercase letter